### PR TITLE
Now user profile can be opened from public chat

### LINKED ISSUE #5184: Add possibility to navigate to user profile from public chat and go back
### User Story

As a user, I want to initiate chat  with public chat participants

### Description

*Type*: Feature
*Summary*: user should be able to open profile public chat member by clicking on his profile picture and go back.

#### How it is implemented in mobile

[comment]: # (Describe what you expected to happen.)
When user clicks on user pic of any public chat participant, he is redirectied to:
lit
The arrow in left top corner is redirecting user back to public chat.

### Solution
[comment]: # (Please summarise the solution and provide a task list on what needs to be fixed.)
*Summary*: 

- [ ] add redirect to user profile from public chat
- [ ] add way for user to get back to public chat

### PR DESCRIPTION
**Pull request by vkjr**
fixes #5184 

### Summary:
Now user profile can be opened from public chat on desktop

### Steps to test:
- Open Status
- go to public chat, like `#status`
- open profile of any user, check that it opens and shows user information

status: ready

### CODE REVIEW

**Review by goranjovic — APPROVED** [review]

File: src/status_im/ui/screens/desktop/main/chat/events.cljs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+(ns screens.desktop.main.chat.events
+  (:require [re-frame.core :as re-frame]
+            [status-im.ui.screens.navigation :as navigation]
+            [status-im.utils.handlers :as handlers]))
+
+(defn show-profile-desktop [identity {:keys [db] :as cofx}]
+  (navigation/navigate-to-cofx :chat-profile nil {:db (assoc db :contacts/identity identity)}))
```
Comment (flexsurfer) [inline src/status_im/ui/screens/desktop/main/chat/events.cljs]: better to use `merge-fx`

**Review by flexsurfer — CHANGES_REQUESTED** [review]

**Review by vitvly — APPROVED** [review]

**Review by maxhora — APPROVED** [review]

**Comment by vkjr** [comment]
@flexsurfer, thanks, fixed.

**Review by flexsurfer — APPROVED** [review]

**Comment by churik** [comment]
@vkjr 
this PR cannot be built.
https://ci.status.im/job/status-react/job/pull%20requests/view/change-requests/job/PR-5546/3/console

```
Aug 20, 2018 8:15:05 AM com.google.javascript.jscomp.LoggerErrorManager printSummary
WARNING: 1 error(s), 0 warning(s)
ERROR: JSC_MISSING_PROVIDE_ERROR. required "screens.desktop.main.chat.events" namespace never provided at /Users/jenkins/workspace/react_pull_requests_PR-5546-FSANQVJBHVI4YTKWLTQJXCXBJYV25KZ63DQY7NFLW522YRJHNZJQ/target/desktop-prod/status_im/ui/screens/desktop/main/chat/views.js line 21 : 0
[31mCompiling ["index.desktop.js"] failed.
```

Can you please help to fix it?

**Comment by churik** [comment]
1. Cannot navigate to user profile when there are several messages in a row (should be last in chat)
Steps:
- join to any chat where several last messages are sent in a row from one user (21/08/18 it is `#tery` and `status-desktop`)
- click on user pic to navigate to user profile

* Expected: *
navigated to user profile

* Actual: *
cannot navigate when clicking on user picture
dfsfd

* Video: * http://take.ms/I0jFN
* Log: * [StatusIm.txt](https://github.com/status-im/status-react/files/2306728/StatusIm.txt)


<blockquote><div>Monosnap screenshot tool</div><div><strong><a href="https://monosnap.com/file/ZQpuZrtZ0vu7ZnVapukOK4RcKUTyZk">File "screencast 2018-08-21 17-29-28.mp4"</a></strong></div><div>Monosnap — the best tool to take & share your screenshots.</div></blockquote>

**Comment by vkjr** [comment]
@churik, thanks for finding. Strange bug, not always reproducible with steps you described, but quite often, investigating...

**Comment by vkjr** [comment]
@churik, I investigated a bit and suspect that problem is not on clojure side, probably something in react-native-desktop. I would suggest to merge this PR and create separate issue for a problem. Because functionality is quite demanded and issue itself is not that critical.
What do you think?

**Comment by churik** [comment]
Yes, OK. Can you create separate issue on react-native side and link it to PR or how we can track the progress?

**Comment by churik** [comment]
@vkjr I retest it one more time and noticed one more strange behavior - when you scroll through history sometimes navigating just stop working and I faced two times after constant switching that app stops responding.
So it works +/- stable only for messages from today and when they are not sent in row from one user.

Video: http://take.ms/SDHC4
Log: [StatusIm.txt](https://github.com/status-im/status-react/files/2309370/StatusIm.txt)

@chadyj is it acceptable?


<blockquote><div>Monosnap screenshot tool</div><div><strong><a href="https://monosnap.com/file/VwAgqP8Ek5gLeHLoaDttbQLDBBcEz8">File "screencast 2018-08-22 10-55-40.mp4"</a></strong></div><div>Monosnap — the best tool to take & share your screenshots.</div></blockquote>

**Comment by vkjr** [comment]
@churik, unfortunately I can't create react-native issue for now, because I don't know exact reason and don't have minimal example to reproduce it. I can create status-react issue and work on it and add react-native-desktop issue if will figure out it.

**Comment by vkjr** [comment]
But I'm pretty sure that problem is not in the clojurescript code that is going to be commited in this PR. So if we commit it in 'develop', fixing react-native-desktop side can automatically fix the issue
.

**Comment by chadyj** [comment]
If it's a regression in the PR, let's fix before merging.

If not, lets merge and create a new issue.

**Comment by churik** [comment]
Reported in https://github.com/status-im/status-react/issues/5597

**Pull request merged**